test/unit/bank-redirect.test.js

```diff
@@ -245,4 +245,138 @@ describe('Recurly.BankRedirect', function () {
       });
     });
   });
+
+  describe('sofort', function () {
+    describe('loadCountries', function () {
+      it('emits the countries without error for a successful api request', function (done) {
+        this.bankRedirect.on('error', () => assert.fail);
+        this.bankRedirect.on('countries', countries => {
+          assert.ok(countries);
+          done();
+        });
+
+        this.bankRedirect.loadCountries();
+      });
+
+      it('attach the countries to a select element', function (done) {
+        fixture('selectLists', 'country_code');
+        const $select = testBed().querySelector('#country_code');
+
+        this.bankRedirect.on('error', () => assert.fail);
+        this.bankRedirect.on('countries', () => {
+          assert.ok(new RegExp([
+            '<select (id|name)="country_code" (id|name)="country_code">',
+            '<option value="AT">Austria</option>',
+            '<option value="BE">Belgium</option>',
+            '<option value="DE">Germany</option>',
+            '<option value="IT">Italy</option>',
+            '<option value="ES">Spain</option>',
+            '<option value="NL">The Netherlands</option>',
+            '</select>'
+          ].join('')).test($select.outerHTML));
+          done();
+        });
+
+        this.bankRedirect.loadCountries('#country_code');
+      })
+
+      it('attach the countries to a select element after cleanup the select', function (done) {
+        const list = [
```

Comment:
The idea of this test is to make sure whatever `<select>` element the user provide, we clean up its `<options>` before put the countries, so, I'd rather initialize the `<select>` with different `<options>`.